### DISCUSSION TITLE
💗 I just recieved an overwhelming donation

### DISCUSSION THREAD

**Discussion by GiorgosXou** [GiorgosXou/TUIFIManager#102] [Announcements]
Please, check @Naheel-Azawy's profile and his work, show him some love. He just did a really significant donation. [He reenforced the ability of my existance and therefore the existance of this project.](https://github.com/GiorgosXou/TUIFIManager/discussions/71) Therefore, I promise I won't abandon this project soon and that this is just the begining.
 
 Thanks.

**Comment by Naheel-Azawy** [comment]
It's my pleasure, you deserve much more!

**Comment by Sigmanificient** [comment]
Congrats, you deserved it!
